autosklearn/estimators.py

```diff
@@ -168,6 +169,10 @@ def __init__(self,
             This is an advanced feature. Use only if you are familiar with
             `SMAC <https://automl.github.io/SMAC3/stable/index.html>`_.
 
+        log_config : dict, optional (None)
+            dictionary object specifying the logger configuration. If None,
+            the default logging.yaml file is used.
```

Comment:
Could you please provide the full path of the logging.yaml file (relative to the package root)?